src/java/htsjdk/samtools/CRAMFileWriter.java

```diff
@@ -46,8 +46,9 @@
 
 @SuppressWarnings("UnusedDeclaration")
 public class CRAMFileWriter extends SAMFileWriterImpl {
-    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -2;
+    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -3;
     static int DEFAULT_RECORDS_PER_SLICE = 10000;
+    static int SWITCH_TO_MULTIREF_IF_MORE_THAN = 1000;
```

Comment:
this is to prevent many tiny containers, like a threshold when to switch to multiref rather than start another container. I'm changing the constant name and the code to reflect this better.